app/views/publishers/vacancies/job_applications/index.html.slim

```diff
@@ -5,38 +5,10 @@
 .govuk-grid-row
   .govuk-grid-column-full
     - if @vacancy.within_data_access_period?
-      - applications = @job_applications.group_by(&:status)
-      - new_ones = applications.fetch("submitted", []) + applications.fetch("reviewed", [])
-      - rejected = applications.fetch("unsuccessful", [])
-      - shortlisted = applications.fetch("shortlisted", [])
-      - interviewing = applications.fetch("interviewing", [])
       = govuk_tabs do |tabs|
-        / set id property so that anchors are '#shortlisted' rather than '#shortlisted-4'
-        - tabs.with_tab(label: "All (#{@job_applications.size})", id: "all") do
-          - if @job_applications.any?
-            = render "candidates", form: @form, vacancy: @vacancy, candidates: @job_applications, heading: "All Applications", multi_select: "all", origin: :all
-          - else
-            = render EmptySectionComponent.new title: t(".no_applicants")
-        - tabs.with_tab(label: "New (#{new_ones.size})", id: "submitted")
-          - if new_ones.any?
-            = render "candidates", form: @form, vacancy: @vacancy, candidates: new_ones, heading: "New Applications", multi_select: "new", origin: :submitted
-          - else
-            = render EmptySectionComponent.new title: t(".no_new")
-        - tabs.with_tab(label: "Not Considering (#{rejected.size})", id: "not_considering")
-          - if rejected.any?
-            = render "candidates", form: @form, vacancy: @vacancy, candidates: rejected, heading: "Not being considered", multi_select: "rej", origin: :not_considering
-          - else
-            = render EmptySectionComponent.new title: t(".no_rejected")
-        - tabs.with_tab(label: "Shortlisted (#{shortlisted.size})", id: "shortlisted")
-          - if shortlisted.any?
-            = render "candidates", form: @form, vacancy: @vacancy, candidates: shortlisted, heading: "Shortlisted Applications", multi_select: "short", origin: :shortlisted
-          - else
-            = render EmptySectionComponent.new title: t(".no_shortlisted")
-        - tabs.with_tab(label: "Interviewing (#{interviewing.size})", id: "interviewing")
-          - if interviewing.any?
-            = render "candidates", form: @form, vacancy: @vacancy, candidates: interviewing, heading: "Interviewing Applications", multi_select: "inter", origin: :interviewing
-          - else
-            = render EmptySectionComponent.new title: t(".no_interviewing")
+        - @form.tabs.each do |tab_name, count|
+          - tabs.with_tab(label: t(".tab.#{tab_name}.label", count:), id: tab_name)
+            = render "tab_panel", form: @form, vacancy: @vacancy, tab_name:
```

Comment:
I'm not sure why this is an improvement? Each tab is potentially different, so this feels like a backward step. Adding the 'job offered' tab at the bottom of this feels like a better solution